service/src/main/kotlin/fi/espoo/evaka/pis/EmployeeQueries.kt

```diff
@@ -221,7 +229,14 @@ SELECT
         FROM daycare_acl acl
         JOIN daycare d ON acl.daycare_id = d.id
         WHERE acl.employee_id = employee.id
-    ) AS daycare_roles
+    ) AS daycare_roles,
+    (
+        SELECT jsonb_agg(json_build_object('daycareId', d.id, 'daycareName', d.name, 'groupId', dg.id, 'groupName', dg.name))
```

Comment:
`json_build_object` -> `jsonb_build_object`. There's no need to generate *textual* JSON intermediates